{"name": "test_sea_file_login[seafile@admin.com-admin666-400-Unable to login with provided credentials]", "status": "passed", "steps": [{"name": "接口1（登录接口）的用例", "status": "passed", "attachments": [{"name": "步骤01", "source": "56ca6202-2270-4660-8f9e-acf6d6518411-attachment.attach"}, {"name": "步骤02", "source": "f09f4ac6-fde9-4879-bbcf-3a923263d47b-attachment.attach"}, {"name": "步骤03", "source": "bcb1c790-d0fd-4b9c-bebf-32461aa6128d-attachment.attach"}, {"name": "步骤04", "source": "9e39d3d7-8875-4ac8-9bfc-36dd115df131-attachment.attach"}], "parameters": [{"name": "username", "value": "'seafile@admin.com'"}, {"name": "password", "value": "'admin666'"}, {"name": "code", "value": "'400'"}, {"name": "msg", "value": "'Unable to login with provided credentials'"}], "start": 1749818207251, "stop": 1749818207270}], "attachments": [{"name": "log", "source": "48902156-ec48-4c1f-be48-6123d2c41d3c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'seafile@admin.com'"}, {"name": "password", "value": "'admin666'"}, {"name": "code", "value": "'400'"}, {"name": "msg", "value": "'Unable to login with provided credentials'"}], "start": 1749818207251, "stop": 1749818207270, "uuid": "8438f181-694e-4662-9902-ff76f25f39bb", "historyId": "e7bc4e0d840b12f94864f4016fc54c55", "testCaseId": "de8b2b1b22432fbc4ae3a40e7eff706d", "fullName": "test_case.test_07#test_sea_file_login", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_07"}, {"name": "host", "value": "stu001243"}, {"name": "thread", "value": "17232-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_07"}]}